clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -102,7 +102,8 @@ public void testSerialization() throws Exception {
 
         checkSerialization(createUpdateMetadataRequest(0, null), 0);
         checkSerialization(createUpdateMetadataRequest(0, null).getErrorResponse(0, new UnknownServerException()), 0);
-        checkSerialization(createUpdateMetadataRequest(1, "rack1"), 1);
+        checkSerialization(createUpdateMetadataRequest(1, null), 1);
```

Comment:
The rack field is needed here to test that v1 serialization is not broken even if we construct the broker with rack.